openshift/template.yaml

```diff
@@ -40,7 +40,6 @@ objects:
           value: $(tt.params.payload)
         pipelineRef:
           name: cad-checks-pipeline
-        serviceAccountName: cad-sa
```

Comment:
which SA will be used here ? If it ever needs a specific RBAC, it would be good to keep a dedicated one. I don't think so, but just checking

Reply:
a quick test showed that the `pipeline` serviceaccount would be used by default. I don't think your pipeline has any need on the local cluster by default, so I'd use a custom SA there maybe.

Reply:
Yeah, I can revert back to your suggestion, thanks!

Reply:
I am confused now :D @patjlm  didn't you suggest that we can just leave serviceAccountName empty and use the pipeline SA? Why the sudden revert? (Just want to understand this, I am fine with both decisions :) )

Reply:
ok, I understand

Reply:
I don't mind leaving this as is for now. But the least-privilege principle is definitely something you should target to implement

Reply:
I agree

Reply:
@georgettica  do you want to fix the SA line? I was wrong ^^ I think you wanted to leave the custom SA during our meeting and I told you to remove it. Sorry :/